components/contributors/InfoCard.tsx

```diff
@@ -63,25 +79,20 @@ export default function InfoCard({
                 {contributor.name}
               </h3>
             </Link>
-            <p className="text-sm text-secondary-400 md:text-base">
+            <p className="text-sm text-gray-400 md:text-base">
```

Comment:
Ensure conflicts are resolved properly. We are no longer using `gray` shade directly as secondary colors are configurable. Use `secondary` shade instead.